yb-voyager/cmd/importData.go

```diff
@@ -1005,9 +1005,26 @@ func setTargetSchema(conn *pgx.Conn) {
 		// No need to set schema if importing in the default schema.
 		return
 	}
+	target.Schema = strings.ToLower(target.Schema)
```

Comment:
Is it possible to do this conversion somewhere else in the code. Where it is more intuitive, at some upper layer of the code.
This function will be called multiple times and everytime this conversion will be done.

Reply:
Maybe somewhere in `init()` of the commands